pontoon/base/static/js/theme-switcher.js

```diff
@@ -0,0 +1,79 @@
+$(function () {
+  function getSystemTheme() {
+    if (
+      window.matchMedia &&
+      window.matchMedia('(prefers-color-scheme: light)').matches
+    ) {
+      return 'dark';
+    } else {
+      return 'light';
+    }
+  }
+
+  function applyTheme(newTheme) {
+    if (newTheme === 'system') {
+      newTheme = getSystemTheme();
+    }
+    $('body')
+      .removeClass('dark-theme light-theme system-theme')
+      .addClass(`${newTheme}-theme`)
+      .data('theme', newTheme);
```

Comment:
We should move this line to the `ajax.success` function. `applyTheme()` is also called from the event handler, which should not set the data attribute.

Reply:
And, we should also set the `body[data-theme]` attribute in the templates, otherwise changing the system theme in the OS doesn't work.